lib/decidim/maintainers_toolbox/action_backporter.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+require_relative "github_manager/querier"
+require_relative "github_manager/poster"
+require_relative "git_backport_manager"
+
+module Decidim
+  module MaintainersToolbox
+    class ActionBackporter
+      class InvalidMetadataError < StandardError; end
+
+      # @param token [String] token for GitHub authentication
+      # @param pull_request_id [String] the ID of the pull request that we want to backport
+      # @param exit_with_unstaged_changes [Boolean] wheter we should exit cowardly if there is any unstaged change
+      def initialize(token: , pull_request_id: ,exit_with_unstaged_changes: )
```

Comment:
This should be detected in the linter :/ 
```suggestion
      def initialize(token:, pull_request_id:, exit_with_unstaged_changes: )
```